### PR TITLE
Possibility to add whitelisted paths that does not need password protection

### PR DESCRIPTION
**Pull request by estien**

### CODE REVIEW

File: AuthishModule/ValidationService.cs [hunk 1]
```diff
@@ -7,6 +9,7 @@ static class ValidationService
     {
         private static readonly ILog Log = LogManager.GetLogger(typeof(AuthishHandler));
         private static readonly string AuthishPassword = ConfigurationManager.AppSettings["AuthishPassword"];
+        private static readonly string[] AuthishWhitelistedPaths = (ConfigurationManager.AppSettings["AuthishWhitelistedPaths"] ?? "").Split(',');
```
Comment (thomassvensen) [inline AuthishModule/ValidationService.cs]: It's useful to do a "trim" on the indvidual parts after splitting on ",". You will often have people having configuration with something like "path1, path2", which would give " path2" on the second item, and that would never match.

**Comment by thomassvensen** [comment]
Excellent stuff! Thank you!

File: AuthishModule/ValidationService.cs (hunk 1 above)
Comment (estien) [inline AuthishModule/ValidationService.cs]: Agreed. You were too fast on merging, so I couldn't fix it in time :facepunch:

Comment (thomassvensen) [inline AuthishModule/ValidationService.cs]: Right, sorry, my fault (not on top of the PR workflow after too much PowerPoint work lately ;-))

**Pull request merged**